integration_testing/features/coach/coach-class-reports-lessons.feature

```diff
@@ -92,6 +94,33 @@ Feature: Lessons subtab
       # Clarify conditions for *Needs help*
       Then their *Progress* column states *Needs help*
 
+  Scenario: View the resource report page by groups
+    Given that I am on the <resource> report page
```

Comment:
Should add an assumption that the classroom has groups.

I totally forgot to test this, but what happens if the classroom has no groups and you click the checkbox?

Reply:
> I totally forgot to test this, but what happens if the classroom has no groups and you click the checkbox?

Basically, the same logic is used all the time => in this particular case when there are no groups at all, all learners will be displayed in "Ungrouped learners".

<img width="934" alt="Screenshot 2019-05-14 at 18 45 45" src="https://user-images.githubusercontent.com/13509191/57716278-e912fe80-7678-11e9-951b-22c53b6b3915.png">

Reply:
Agree that it makes sense to be more explicit on this in Gherkin stories, will update.

Reply:
Updated + I also extracted empty groups scenarios to their own stories to make things a bit more readable.